packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Fixed
   * Reduced number of unnecessary rerenders in the `terra-worklist-data-grid` component.
+  * Updated cells to honor the value of props passed by consumers.
```

Comment:
We should move this to the `changed` section.

Reply:
Well, this was kind of a defect, but I can move it if we feel that is best.